src/commands/init.js

```diff
@@ -69,8 +69,18 @@ const choices = [
 ];
 const featureNames = choices.map((choice) => choice.value);
 
+const DESCRIPTION = `Interactively configure the current directory as a Firebase project or initialize new features in an already configured Firebase project directory.
+
+This command will create or update 'firebase.json' and '.firebaserc' configuration files in the current directory. 
```

Comment:
I think there's now a warning before it updates existing files, correct?  We might want to mention something about that.  Maybe something like:

This command creates 'firebase.json' and '.firebaserc' configuration files in the current directory. If these files already exist, this command requests confirmation before overwriting any configuration in the file.

Reply:
Actually not this command, `firebase init` doesn't ask.

Reply:
ugh... you're right.  I wish the "warning" was a bit more noticeable.  We can just leave the line as you have it since there's no real speedbump.